addons/web/static/src/core/dialog/dialog.scss

```diff
@@ -1,52 +1,13 @@
-.o_dialog {
-  > .modal {
-    display: block;
-  }
-}
-
-.modal.o_technical_modal {
-  .modal-content {
-    border-radius: 0;
-
-    .modal-header .modal-title {
-      word-break: break-word;
-    }
-
-    .modal-footer {
-      flex-wrap: wrap;
-      text-align: left;
-      justify-content: flex-start;
+.modal.o_technical_modal .modal-footer {
+    button {
+        margin: map-get($spacers, 1);
```

Comment:
could we use gap instead? I know it's not supported for all iOS version but in this case, it's not a big deal.

Reply:
@adr-odoo , done 👍